homeassistant/components/homematicip_cloud/sensor.py

```diff
@@ -176,6 +180,29 @@ def unit_of_measurement(self):
         return TEMP_CELSIUS
 
 
+class HomematicipThermostatTemperatureSensor(HomematicipGenericDevice):
```

Comment:
Why is this a separate sensor and not part of a climate entity?

Reply:
Climate entity does not represent a device. If you have several climate devices (Wall thermostat, flour heating radio thermostat)  assigned to a room, then the Heating-Group (climate entity) controls the behaviour of all related device and only the temperature of the leading thermostat (Wall-Thermostat) is shown. 
This is the individual value of the Radiator Thermostat.

Reply:
You mean that there are multiple temperature sensors involved in the thermostat and the climate entity only uses one of them in `current_temperature`? So the sensors we're adding here are not the one already used in the climate entity?

Reply:
There are multiple temperature sensors in a room and a hmip heating_group represents the climate devices of a room and a  only a hmip heating_group is used for the climate entity.

I started again a real life test after your first reply, if these sensors are really independant, when both are part of the same room.  
Result: There is sometimes a small diffrence, but that might depend on when the device sends an update. Might be that i misinterpreted that there are real diffrences this morning.

In this case we don't need the additional sensor.
As a result we could also get rid of HomematicipTemperatureSensor which is also part of the climate entity. But this would be a breaking change.
I have long term measures for HomematicipTemperatureSensor and climate.current_temparature and they are identical.
@mxworm: What do you think about removing HomematicipTemperatureSensor too?

I remove the HomematicipThermostatTemperatureSensor now.